bin/pt-eustack-resolver

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env perl
+
+# This program is part of Percona Toolkit: http://www.percona.com/software/
+# See "COPYRIGHT, LICENSE, AND WARRANTY" at the end of this file for legal
+# notices and disclaimers.
+
+use strict;
+use Data::Dumper;
+
+my $mmap;
+my $FH;
+
+sub load_mapping {
+   my ($pid)= @_;
+
+   if ($pid =~ /^[0-9]+$/) {
+      open $FH, '<', "/proc/$pid/maps"
+         or die "Failed to open /proc/$pid/maps: $!\n";
```

Comment:
open my $FH

Reply:
This would not work, because this code is in the scope of the if statement.